aws/index.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
It seems we need the full path for the metadata file 
This will help the AVD generator